bundle.js

```diff
@@ -0,0 +1,110 @@
+(() => {
+  var __getOwnPropNames = Object.getOwnPropertyNames;
+  var __commonJS = (cb, mod) => function __require() {
+    return mod || (0, cb[__getOwnPropNames(cb)[0]])((mod = { exports: {} }).exports, mod), mod.exports;
+  };
+
+  // newsView.js
+  var require_newsView = __commonJS({
+    "newsView.js"(exports, module) {
+      var NewsView2 = class {
+        constructor(model2, api2) {
+          this.model = model2;
+          this.mainContainerEl = document.querySelector("#main-container");
+          this.inputEl = document.querySelector("#inputEl");
+          this.btnEl = document.querySelector("#btnEl");
+          this.btnEl.addEventListener("click", () => {
+            const userSearch = this.inputEl.value;
+            console.log(userSearch);
+            this.inputEl.value = "";
+            api2.searchHeadlines(userSearch, (data) => {
+              this.model.setHeadlines(data);
+              this.displayHeadlines(data);
+            });
+          });
+        }
+        displayHeadlines(data) {
+          const newsArticles = [];
+          data.forEach((headline) => {
+            const headlineEl = document.createElement("div");
+            headlineEl.className = "headline";
+            const hrefEl = document.createElement("a");
+            hrefEl.className = "article-link";
+            hrefEl.href = headline.webUrl;
+            hrefEl.innerText = headline.webTitle;
+            const imgEl = document.createElement("img");
+            imgEl.className = "article-img";
+            imgEl.src = headline.fields.thumbnail;
+            headlineEl.append(imgEl);
+            headlineEl.append(hrefEl);
+            newsArticles.push(headlineEl);
+          });
+          this.mainContainerEl.replaceChildren(...newsArticles);
+        }
+      };
+      module.exports = NewsView2;
+    }
+  });
+
+  // newsModel.js
+  var require_newsModel = __commonJS({
+    "newsModel.js"(exports, module) {
+      var NewsModel2 = class {
+        constructor() {
+          this.headlines = [];
+        }
+        getHeadlines() {
+          return this.headlines;
+        }
+        addHeadline(text) {
+          this.headlines.push(text);
+        }
+        setHeadlines(headlines) {
+          this.headlines = [...headlines];
+        }
+      };
+      module.exports = NewsModel2;
+    }
+  });
+
+  // apiKey.js
+  var require_apiKey = __commonJS({
+    "apiKey.js"(exports, module) {
+      var guardianApi = "49fcc868-44be-47ad-a97f-fe49abdd7bc2";
+      module.exports = guardianApi;
+    }
+  });
+
```

Comment:
I saw someone post about the key showing up here, but as it remains untackled for me I lack any constructives I'm afraid.